src/LinearSolvers/LinearSolvers.jl

```diff
@@ -21,27 +21,33 @@ module LinearSolvers
 	end
 	
 	# check if Pardiso is installed
+	const minPardisoVersion = VersionNumber(0,1,2)
 	hasPardiso = false
 	try
 		using Pardiso
 		hasPardiso = true
+		if myid()==1
+		  vPardiso = Pkg.installed("Pardiso")
+		  if vPardiso < minPardisoVersion
+		    warn("jInv Pardiso support requires Pardiso.jl version 0.1.2 or greater. Pardiso support will not be loaded")
+		    hasPardiso = false
+		  end
+		end
 	catch
 	end
 	
 	# check if ParSPMatVec is available
 	hasParSpMatVec = false
 	const minVerParSpMatVec = VersionNumber(0,0,1)
-	try 	
```

Comment:
should we do the same as we do for Pardiso (checking the version on main worker)?